src/pages/overview/releasenotes.md

```diff
@@ -185,6 +185,13 @@ Upgrading to the latest SDK should not break existing applications.
 
 ## Change history
 
+### Server Side Release (August, 2023; server side release)
+
```

Comment:
Why is server side repeated here? Also, we're (mostly) APIs - do we need to say 'server side'?